stubs/jsonschema/jsonschema/_types.pyi

```diff
@@ -12,7 +12,7 @@ def is_string(checker, instance): ...
 def is_any(checker, instance): ...
 
 class TypeChecker:
-    def __init__(self, type_checkers: Mapping[str, Callable[[object], bool]] = {}) -> None: ...
+    def __init__(self, type_checkers: Any) -> None: ...
```

Comment:
Pyright was complaining about the `= {}` part. Using the following should work:

```suggestion
    def __init__(self, type_checkers: Mapping[str, Callable[[object], bool]] = ...) -> None: ...
```

(I did not check the implementation yet, so I'm not sure this is correct, though.)

Reply:
Thank you! I'm still trying to figure out the best way to annotate that jsonschema uses a pmap there. So the Mapping[] bits are not exactly right.